src/bio.c

```diff
@@ -66,10 +58,21 @@ static unsigned int bio_job_to_worker[] = {
     [BIO_COMP_RQ_LAZY_FREE]  = 2
 };
 
-static pthread_t bio_threads[BIO_WORKER_NUM];
-static pthread_mutex_t bio_mutex[BIO_WORKER_NUM];
-static pthread_cond_t bio_newjob_cond[BIO_WORKER_NUM];
-static list *bio_jobs[BIO_WORKER_NUM];
+typedef struct {
+    const char *const bio_worker_title;
+    pthread_t bio_thread_id;
+    pthread_mutex_t bio_mutex;
+    pthread_cond_t bio_newjob_cond;
+    list *bio_jobs;
+} bio_worker_data;
+
+static bio_worker_data bio_workers[] = {
+    {"bio_close_file"},
+    {"bio_aof"},
+    {"bio_lazy_free"}
+};
+#define BIO_WORKER_NUM (sizeof bio_workers / sizeof *bio_workers)
```

Comment:
please move these to before bio_job_to_worker, so that we can avoid some changes.